src/core/opamVersionCompare.ml

```diff
@@ -15,28 +15,13 @@
 let is_digit = function
   | '0'..'9' -> true
   | _ -> false
-;;
 
 (* [skip_while_from i f w m] yields the index of the leftmost character
  * in the string [s], starting from [i], and ending at [m], that does
  * not satisfy the predicate [f], or [length w] if no such index exists.  *)
 let rec skip_while_from i f w m =
-  if i = m then i
+  if Int.equal i m then i
   else if f w.[i] then skip_while_from (i + 1) f w m else i
-;;
```

Comment:
The cosmetic change should be in a separate commit

Reply:
done